docs/sources/alerting/configure-notifications/template-notifications/images-in-notifications.md

```diff
@@ -118,6 +118,7 @@ Grafana supports a wide range of contact points with varied support for images i
 ## Limitations
 
 - This feature is not supported in Mimir or Loki, or when Grafana is configured to send alerts to other Alertmanagers such as the Prometheus Alertmanager.
+- This feature is not supported when using custom modified templates in notifications.
```

Comment:
I'd clarify that this applied to emails. Other integrations should attach images even when using custom templates.
```suggestion
- This feature is not supported when using custom templates in email notifications.
```